server/src/main/java/org/elasticsearch/index/store/LuceneFilesExtensions.java

```diff
@@ -76,7 +76,10 @@ public enum LuceneFilesExtensions {
     // kNN vectors format
     VEC("vec", "Vector Data", false, true),
     VEX("vex", "Vector Index", false, true),
-    VEM("vem", "Vector Metadata", true, false);
+    VEM("vem", "Vector Metadata", true, false),
```

Comment:
@javanna there are new file extensions we need to be aware of. I added them here.